__tests__/app.test.js

```diff
@@ -33,14 +33,48 @@ describe("GET /api/topics", () => {
       });
   });
 });
-describe('GET /api', () => {
-  test('GET:200 sends an object containing description of the endpoints', () => {
+describe("GET /api", () => {
+  test("GET:200 sends an object containing description of the endpoints", () => {
     return request(app)
-    .get("/api")
-    .expect(200)
-    .then((response) => {
-      const endPoints = response.body.parsedEndpoints;
-        expect(endPoints).toEqual(endpointsFile)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const endPoints = response.body;
+        expect(endPoints).toEqual(endpointsFile);
+      });
+  });
+});
+describe("GET /api/articles/:article_id", () => {
+  test("GET:200 sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/7")
+      .expect(200)
+      .then((response) => {
```

Comment:
We also need to be checking the article id to check we have the right article back (7)

Reply:
oh yeah, of course